docs/resource-policy/setup.md

```diff
@@ -0,0 +1,61 @@
+# Setup and Usage
+
+When you want to try NRI Resource Policy, here is the list of things
+you need to do, assuming you already have a Kubernetes\* cluster up and
+running, using either `containerd` or `cri-o` as the runtime.
+
+  * [Install](installation.md) NRI Resource Policy DaemonSet deployment file.
+  * Runtime (containerd / cri-o) configuration
+
+For NRI Resource Policy, you need to provide a configuration file. The default
+configuration ConfigMap file can be found in the DaemonSet deployment yaml file.
+You can edit it as needed.
+
+**NOTE**: Currently, the available policies are a work in progress.
+
+## Setting up NRI Resource Policy
+
+### Using NRI Resource Policy Agent and a ConfigMap
+
+The [NRI Resource Policy Node Agent][agent] can monitor and fetch configuration
+from the ConfigMap and pass it on to NRI Resource Policy plugin.
+By default, it automatically tries to use the agent to acquire configuration,
+unless you override this by forcing a static local configuration using
+the `--force-config <config-file>` option.
+When using the agent, it is also possible to provide an initial fallback for
+configuration using the `--fallback-config <config-file>`. This file is
+used before the very first configuration is successfully acquired from the
+agent.
+
+See the [Node Agent][agent] about how to set up and configure the agent.
+
+
+## Logging and debugging
```

Comment:
Maybe this file should be renamed logging and debugging. Or then move also this section under the agent/configuration doc